src/main/java/athleticli/commands/sleep/ListSleepCommand.java

```diff
@@ -1,28 +1,47 @@
 package athleticli.commands.sleep;
 
+import java.util.logging.Logger;
+
 import athleticli.commands.Command;
 import athleticli.data.Data;
 import athleticli.data.sleep.SleepList;
 import athleticli.ui.Message;
 
 public class ListSleepCommand extends Command {
 
+    private static final Logger LOGGER = Logger.getLogger(ListSleepCommand.class.getName());
+
     /**
      * Lists all the sleep records in the sleep list.
      *
      * @param data The current data containing the sleep list.
      * @return The message which will be shown to the user.
      */
-    public String[] execute (Data data) {
-        SleepList sleepList = data.getSleeps();
-        if (sleepList.size() == 0) {
+    public String[] execute(Data data) {
+        LOGGER.info("Executing ListSleepCommand");
+        SleepList sleeps = data.getSleeps();
+        final int size = sleeps.size();
+        assert size >= 0 : "Sleep list size cannot be negative";
```

Comment:
`size()` is a function provided by the Java standard library and will not return a negative number unless an error occurs in the standard library.